aten/src/ATen/native/native_functions.yaml

```diff
@@ -1391,6 +1391,26 @@
     CPU, CUDA: div_out
     SparseCPU, SparseCUDA: div_out_sparse_zerodim
 
+- func: div.Tensor_mode(Tensor self, Tensor other, str rounding_mode) -> Tensor
+  use_c10_dispatcher: full
+  variants: function, method
+  dispatch:
+    CPU, CUDA: div
+    SparseCPU, SparseCUDA: div_sparse
```

Comment:
I think we can remove sparse support for rounding modes from this PR.

Reply:
Moved sparse changes higher up the stack.